snuba/web/rpc/v1/resolvers/R_eap_spans/common/sampling_in_storage_util.py

```diff
@@ -132,16 +139,14 @@ def _get_target_tier(
 
         target_tier = _get_most_downsampled_tier()
         estimated_target_tier_bytes_scanned = (
-            most_downsampled_query_bytes_scanned
-            * cast(int, DOWNSAMPLING_TIER_MULTIPLIERS.get(target_tier))
+            most_downsampled_query_bytes_scanned * _get_multiplier(target_tier)
```

Comment:
It seems like it may be possible for this function to return the 512 tier since that tier is still in the list of tiers from which this function picks its return value. Maybe this doesn't happen in practice because of how the logic works, but it's not immediately obvious.

We could just remove the 512 tier from that list, but I'd imagine we considered and rejected that approach. What's the reason?